package.json

```diff
@@ -37,58 +70,30 @@
     "stylelint-config-standard": "^17.0.0",
     "supertest": "^3.0.0",
     "testpilot-ga": "^0.3.0",
-    "uglifyify": "^4.0.3"
+    "uglifyify": "^4.0.3",
```

Comment:
Here is a mistake in our dependencies. **babel-polyfill** is used in production code, so I think it should not be placed in **devDependecies** but **dependencies**. So do **jquery**, **testpilot-ga** and others libraries required in production code. I'll send another PR to fix it.